### PR TITLE
fix broken link

### PR DESCRIPTION
**Pull request by casperdcl**

### CODE REVIEW

**Comment by shcheklein** [comment]
@casperdcl thanks!

File: static/docs/install/index.md [hunk 1]
```diff
@@ -16,5 +16,5 @@ Please double check that you don't already have DVC (for example running
 - You can [install a stable pre-release](/doc/install/pre-release) of DVC to
   stay ahead of official releases.
 
-- Please also check out these 3rd-party tool
-  [plugins](/doc/user-guide/install/plugins), which might be useful.
+- Please also check out these 3rd-party tool [plugins](/doc/install/plugins),
```
Comment (jorgeorpinel) [inline static/docs/install/index.md]: Hmmm this makes me wonder if any others were broken. Let me look up the PR that extracted the install section out of the user guide...

**Comment by jorgeorpinel** [comment]
> darn yarn

Lol @casperdcl do you have the pre-commit hook setup? See https://dvc.org/doc/user-guide/contributing/docs#development-environment

**Comment by casperdcl** [comment]
I committed directly from GitHub

**Comment by shcheklein** [comment]
@casperdcl @jorgeorpinel yep, that's why we need to install restyled :) so, that it's easy to edit it directly using Github as CMS admin :)

File: static/docs/install/index.md (hunk 1 above)
Comment (jorgeorpinel) [reply static/docs/install/index.md]: Should be all good per the changes in https://github.com/iterative/dvc.org/pull/734/files?file-filters%5B%5D=.json#diff-19166edb015cd737a10d78c350d18229

**Pull request merged**